ouroboros-consensus/src/Ouroboros/Consensus/Storage/ChainDB/Impl/Types.hs

```diff
@@ -627,6 +643,23 @@ data TraceAddBlockEvent blk =
     -- This is done for all blocks from the future each time a new block is
     -- added.
   | ChainSelectionForFutureBlock (RealPoint blk)
+
+    -- | The ChainDB has finished processing an instruction to add a block.
+    --
+    -- If the block was relevant, valid, new, and so on, then this instruction
```

Comment:
Do we need the "and so on" part?

Reply:
Thanks for calling out a wishy-washy word. I simplified the comment.